File: v1/implementation/Turtle.cpp
// Turtle.cpp

#include "../header/Turtle.h"
#include <iostream>
#include <thread>
#include <chrono>
using namespace std;

Turtle::Turtle(const Point& P): Hewan(TURTLE_ID, TURTLE_MAXAGE)
{
    age = 0;
    status = 1;
    power = TURTLE_BASEPOWER;
    pos = P;
}

File: TubesMakhluk/src/tubesmakhluk/Turtle.java
package tubesmakhluk;

public class Turtle extends Hewan
{
	private static final char TURTLE_ID = 'T';
	private static final int TURTLE_MAXAGE = 240;
	private static final int TURTLE_BASEPOWER = 2;

	public Turtle(final Point P)
	{
		super(TURTLE_ID, TURTLE_MAXAGE);

		age = 0;
		status = 1;
		power = TURTLE_BASEPOWER;
		pos = P;
	}
}